domain-model/src/test/java/uk/gov/hmcts/cmc/domain/models/claimantresponse/ResponseAcceptationTest.java

```diff
@@ -26,44 +26,31 @@ public void shouldBeSuccessfulValidationForValidResponse() {
     }
 
     @Test
-    public void shouldBeInvalidWhenAmountNotPresent() {
+    public void shouldBeValidWhenAmountIsZero() {
```

Comment:
You could have kept an old case changing invalid to valid. This way case where amount is null would have been tested as well.

Reply:
do we really want to test validation on optional (not annotated) ?